pom.xml

```diff
@@ -274,7 +274,13 @@
                         <exclude>**/hbase/data/**</exclude>
                         <exclude>**/kafkazk/data/**</exclude>
                         <exclude>**/wait-for-it.sh</exclude>
-			<exclude>**/*.out</exclude>
+			                  <exclude>**/*.out</exclude>
+                        <exclude>node_modules/**</exclude>
+                        <exclude>node/**</exclude>
```

Comment:
Can we make these a bit less broad?  Maybe only have these run inside of specific directories?